darts/models/regression_ensemble_model.py

```diff
@@ -0,0 +1,100 @@
+"""
+Regression ensemble model
+-------------------------
+"""
+from sklearn.linear_model import LinearRegression
+from typing import Optional, List
+
+from darts.timeseries import TimeSeries
+from darts.models import EnsembleModel, StandardRegressionModel
+from darts.models.forecasting_model import ForecastingModel
+from darts.logging import get_logger, raise_if, raise_if_not
+
+logger = get_logger(__name__)
+
+try:
+    from darts.models.torch_forecasting_model import TorchForecastingModel
+    TORCH_AVAILABLE = True
+except ImportError:
+    logger.warning('Torch not available')
+    TORCH_AVAILABLE = False
+
+
+class RegressionEnsembleModel(EnsembleModel):
+    def __init__(self,
+                 forecasting_models: List[ForecastingModel],
+                 regression_train_n_points: int,
+                 regression_model=LinearRegression(n_jobs=-1, fit_intercept=False)):
+        """
+        Class for ensemble models using a regression model for ensembling individual models' predictions
+        The provided regression model must implement fit() and predict() methods
+        (e.g. scikit-learn regression models)
+
+        Parameters
+        ----------
+        forecasting_models
+            List of forecasting models whose predictions to ensemble
+        regression_train_n_points
+            The number of points to use to train the regression model
+        regression_model
+            Any regression model with predict() and fit() methods (e.g. from scikit-learn)
+            Default: `sklearn.linear_model.LinearRegression(n_jobs=-1, fit_intercept=False)`
+        """
+        super().__init__(forecasting_models)
+
+        # wrap provided regression_model in a StandardRegressionModel (if not already the case)
+        if isinstance(regression_model, StandardRegressionModel):
+            # raise exception if train_n_points value is ambiguous
+            model_train_n_points = regression_model.train_n_points
+            raise_if(model_train_n_points is not None and regression_train_n_points != model_train_n_points,
+                     "Provided StandardRegressionModel.train_n_points parameter doesn't match specified"
+                     " regression_train_n_points parameter.",
+                     logger)
+
+            regression_model.train_n_points = regression_train_n_points
```

Comment:
I guess that's not needed as you've checked above they are the same.

Reply:
There's a catch:

if `model_train_n_points` is `None`, then `regression_train_n_points != model_train_n_points` isn't evaluated – and in that case it makes sense to set `regression_model.train_n_points` to whichever value was specified by `regression_train_n_points`. Then the rest of the class can assume that`regression_model.train_n_points` will be accurate and only use this one.

That's why I've implemented it that way, but maybe it's not the super obvious?

Reply:
OK, then perhaps it'd be nice to add a small comment, so we are not tempted to remove the line later on :)